packages/flutter/test/material/dropdown_form_field_test.dart

```diff
@@ -669,4 +671,53 @@ void main() {
     await tester.pumpAndSettle();
     expect(find.text('Two as an Arabic numeral: 2'), findsOneWidget);
   });
+
+  testWidgets('DropdownButton onTap callback is called when defined', (WidgetTester tester) async {
+    int dropdownButtonTapCounter = 0;
+    String value = 'one';
+    void didChangeValue(String newValue) {
+      value = newValue;
+    }
+
+    Widget build() => buildFormFrame(
+      value: value,
+      onChanged: didChangeValue,
+      onTap: () { dropdownButtonTapCounter += 1; },
```

Comment:
Since you define the function for `onChanged` outside, maybe extract the function into a variable too for consistency.
Same as below.